### ISSUE TITLE
Cannot save single image from sequence?

### ISSUE THREAD

**Issue by cdgriffith** [fdintino/pillow-avif-plugin#1]
First off, thanks for the great plugin, loving the support for the new format!  The fact it includes sequences is amazing!

One thing I just ran into, if I am trying to save a still from a GIF the traditional way it doesn't work.

```
from PIL import Image
import pillow_avif

image = Image.open("test.gif")
image.seek(0)
image.thumbnail((64, 64))

# works
# image.save("test.png")

# saves as sequence 
image.save("test.avif")
```

**Comment by fdintino** [comment]
You'll need to instead do `image.save("test.avif", save_all=True)`, [same as you would](https://pillow.readthedocs.io/en/stable/handbook/image-file-formats.html#saving) when saving an animated gif

**Comment by cdgriffith** [comment]
Right, I am specifically trying NOT to save it animated. Just a single frame from it.

**Comment by cdgriffith** [comment]
For example, if I use this short GIF as "incoming.gif"

![incoming](https://user-images.githubusercontent.com/3275435/108851696-57088b80-75aa-11eb-9922-b0eb57db01cf.gif)


And I want to save a single frame from it, say right back to a `.gif`, but this time not animated. 
```
from PIL import Image
import pillow_avif

image = Image.open("incoming.gif")
image.seek(0)
image.save("D:\\output.gif")
```

Works as expected:
![output](https://user-images.githubusercontent.com/3275435/108851820-78697780-75aa-11eb-9d98-6f8c5e6b50e2.gif)

If I try that same with avif, it is still animated.

```
from PIL import Image
import pillow_avif

image = Image.open("incoming.gif")
image.seek(0)
image.save("D:\\output.avif", qmax=24)
```

(zipped because github hasn't embraced the future yet)
[output.zip](https://github.com/fdintino/pillow-avif-plugin/files/6029215/output.zip)

**Comment by fdintino** [comment]
Oh, thanks for the clarification! That does indeed seem like a bug.

**Comment by fdintino** [comment]
This should be fixed in the 1.0.1 release, which I just uploaded to PyPi. Let me know if you run into any issues.

**Comment by cdgriffith** [comment]
You are fast! Works as expected, thanks!
